src/test_l1_dsFPD.c

```diff
@@ -881,8 +860,17 @@ void test_l1_dsFPD_positive_dsGetFPBrightness (void)
 
```

Comment:
can we check with mini & max values which is defined in the profile file

Reply:
updated